README.md

```diff
@@ -64,7 +64,7 @@ I find the following features to be intresting:
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-    
+    - [Live Demo Link](https://abenezer-addisu.github.io/Portfolio-version/)
 
```

Comment:
- Great Job for deploying your portfolio. Successfully 👍 however you didn’t make you live demo path a link, please make it a link for easy accessibility and better user experiences 👌
![2023-04-13_10-19](https://user-images.githubusercontent.com/52010789/231683831-129a4e63-a2d7-47f8-9429-fb3a2878c134.png)

Reply:
Fixed.